ADAL/src/telemetry/ADTelemetryPiiRules.m

```diff
@@ -30,7 +30,66 @@ @implementation ADTelemetryPiiRules
 
 + (void)initialize
 {
-    _piiRules = @{AD_TELEMETRY_KEY_USER_ID: @YES};
+    _piiRules = @{
+                  AD_TELEMETRY_KEY_AUTHORITY_TYPE: @NO,
+                  AD_TELEMETRY_KEY_AUTHORITY_VALIDATION_STATUS: @NO,
+                  AD_TELEMETRY_KEY_EXTENDED_EXPIRES_ON_SETTING: @NO,
+                  AD_TELEMETRY_KEY_PROMPT_BEHAVIOR: @NO,
+                  AD_TELEMETRY_KEY_RESULT_STATUS: @NO,
+                  AD_TELEMETRY_KEY_IDP: @NO,
+                  AD_TELEMETRY_KEY_TENANT_ID: @YES,
+                  AD_TELEMETRY_KEY_USER_ID: @YES,
+                  AD_TELEMETRY_KEY_START_TIME: @NO,
+                  AD_TELEMETRY_KEY_END_TIME: @NO,
+                  AD_TELEMETRY_KEY_RESPONSE_TIME: @NO,
+                  AD_TELEMETRY_KEY_DEVICE_ID: @YES,
+                  AD_TELEMETRY_KEY_APPLICATION_NAME: @YES,
+                  AD_TELEMETRY_KEY_APPLICATION_VERSION: @NO,
+                  AD_TELEMETRY_KEY_LOGIN_HINT: @YES,
+                  AD_TELEMETRY_KEY_BROKER_VERSION: @NO,
+                  AD_TELEMETRY_KEY_BROKER_PROTOCOL_VERSION: @NO,
+                  AD_TELEMETRY_KEY_BROKER_APP: @YES,
+                  AD_TELEMETRY_KEY_BROKER_APP_USED: @YES,
+                  AD_TELEMETRY_KEY_CLIENT_ID: @YES,
+                  AD_TELEMETRY_KEY_API_ID: @NO,
+                  AD_TELEMETRY_KEY_TOKEN_TYPE: @NO,
+                  AD_TELEMETRY_KEY_RT_STATUS: @NO,
+                  AD_TELEMETRY_KEY_MRRT_STATUS: @NO,
+                  AD_TELEMETRY_KEY_FRT_STATUS: @NO,
+                  AD_TELEMETRY_KEY_IS_SUCCESSFUL: @NO,
+                  AD_TELEMETRY_KEY_CORRELATION_ID: @NO,
+                  AD_TELEMETRY_KEY_IS_EXTENED_LIFE_TIME_TOKEN: @NO,
+                  AD_TELEMETRY_KEY_API_ERROR_CODE: @NO,
+                  AD_TELEMETRY_KEY_PROTOCOL_CODE: @NO,
+                  AD_TELEMETRY_KEY_ERROR_DESCRIPTION: @YES,
+                  AD_TELEMETRY_KEY_ERROR_DOMAIN: @NO,
+                  AD_TELEMETRY_KEY_HTTP_METHOD: @NO,
+                  AD_TELEMETRY_KEY_HTTP_PATH: @YES,
+                  AD_TELEMETRY_KEY_HTTP_RESPONSE_METHOD: @NO,
+                  AD_TELEMETRY_KEY_REQUEST_QUERY_PARAMS: @YES,
+                  AD_TELEMETRY_KEY_USER_AGENT: @YES,
```

Comment:
we need to confirm if user agent is actually PII

Reply:
marked it as PII.

Reply:
My comment was that it's probably not a PII :)

Reply:
The user agent string is typically just the app name, version and some webkit info, not PII.

Reply:
sorry, made a typo. Marked it as NOT PII :)